test/JDBC/expected/IDENTITY-INSERT-CROSS-DB.out

```diff
@@ -0,0 +1,278 @@
+-- tsql
+CREATE DATABASE identity_insert_db
+GO
+
```

Comment:
Can we avoid making this new file IDENTITY-INSERT-CROSS-DB? , instead I suggest moving this to BABEL-IDENTITY.